python/lsst/afw/math/convolveImage.cc

```diff
@@ -24,22 +24,45 @@
 //#include <pybind11/operators.h>
 //#include <pybind11/stl.h>
 
+#include "lsst/afw/math/ConvolveImage.h"
+
 namespace py = pybind11;
 
 using namespace lsst::afw::math;
 
-PYBIND11_PLUGIN(_convolveImage) {
-    py::module mod("_convolveImage", "Python wrapper for afw _convolveImage library");
+template <typename ImageType1, typename ImageType2>
+void declareConvolveByType(py::module & mod) {
+    /* Members */
+    // declarations for convolve overloads go here...
+    mod.def("scaledPlus", (void (*)(ImageType1 &, double, ImageType2 const &, double, ImageType2 const &)) scaledPlus);
+}
 
-    /* Module level */
+template <typename PixelType1, typename PixelType2>
+void declareConvolve(py::module & mod) {
+    using lsst::afw::image::Image;
+    using lsst::afw::image::MaskedImage;
+    using lsst::afw::image::MaskPixel;
+    using lsst::afw::image::VariancePixel;
```

Comment:
Perhaps just ```using lsst::afw::image;``` can be included, otherwise it will be confusing as to which  header file to find objects in the various namespaces.